cvmfs/cvmfs.cc

```diff
@@ -2626,7 +2637,7 @@ static int AltProcessFlavor(int argc, char **argv) {
     return quota::MainCacheManager(argc, argv);
   }
   if (strcmp(argv[1], "__wpad__") == 0) {
-    return download::MainResolveProxyDescription(argc, argv);
+    return download::MainResolveProxyDescription(argc, argv, cvmfs::statistics_);
```

Comment:
Here we should create the statistics class in MainResolveProxyDescription.  This is one of the "special modes" in which the client can be started.  In this particular special mode, it doesn't act as a fuse module but only resolves the proxy definition and prints the result.